srcpkgs/openjdk14/template

```diff
@@ -0,0 +1,201 @@
+disable_parallel_build=yes
+build_options="docs"
+
+# no hotspot JIT for ppc32
+case "$XBPS_TARGET_MACHINE" in
+	ppc64*) ;;
+	ppc*) _use_zero=yes ;;
+esac
+
+if [ -n "$_use_zero" ]; then
+	makedepends+=" libffi-devel"
+	configure_args+=" --with-jvm-variants=zero"
+	case "$XBPS_TARGET_MACHINE" in
+	ppc*) configure_args+=" --with-boot-jdk-jvmargs=-XX:ThreadStackSize=2560";;
+	esac
+fi
+
+case "$XBPS_TARGET_LIBC" in
+	glibc) build_options_default+=" docs";;
+esac
+
+if [ ! "$CROSS_BUILD" ]; then
+	hostmakedepends+=" openjdk13-bootstrap"
+	configure_args+=" --with-boot-jdk=/usr/lib/jvm/openjdk13"
+else
+	hostmakedepends+=" openjdk14"
+	configure_args+=" --with-boot-jdk=/usr/lib/jvm/openjdk14"
+fi
+
+alternatives="
+ java:/usr/bin/java:/usr/lib/jvm/openjdk14/bin/java
+ java:/usr/bin/jjs:/usr/lib/jvm/openjdk14/bin/jjs
+ java:/usr/bin/keytool:/usr/lib/jvm/openjdk14/bin/keytool
+ java:/usr/bin/pack200:/usr/lib/jvm/openjdk14/bin/pack200
+ java:/usr/bin/rmid:/usr/lib/jvm/openjdk14/bin/rmid
+ java:/usr/bin/rmiregistry:/usr/lib/jvm/openjdk14/bin/rmiregistry
+ java:/usr/bin/unpack200:/usr/lib/jvm/openjdk14/bin/unpack200
+
+ jdk:/usr/bin/jaotc:/usr/lib/jvm/openjdk14/bin/jaotc
+ jdk:/usr/bin/jar:/usr/lib/jvm/openjdk14/bin/jar
+ jdk:/usr/bin/jarsigner:/usr/lib/jvm/openjdk14/bin/jarsigner
+ jdk:/usr/bin/java:/usr/lib/jvm/openjdk14/bin/java
+ jdk:/usr/bin/javac:/usr/lib/jvm/openjdk14/bin/javac
+ jdk:/usr/bin/javadoc:/usr/lib/jvm/openjdk14/bin/javadoc
+ jdk:/usr/bin/javap:/usr/lib/jvm/openjdk14/bin/javap
+ jdk:/usr/bin/jcmd:/usr/lib/jvm/openjdk14/bin/jcmd
+ jdk:/usr/bin/jconsole:/usr/lib/jvm/openjdk14/bin/jconsole
+ jdk:/usr/bin/jdb:/usr/lib/jvm/openjdk14/bin/jdb
+ jdk:/usr/bin/jdeprscan:/usr/lib/jvm/openjdk14/bin/jdeprscan
+ jdk:/usr/bin/jdeps:/usr/lib/jvm/openjdk14/bin/jdeps
+ jdk:/usr/bin/jhsdb:/usr/lib/jvm/openjdk14/bin/jhsdb
+ jdk:/usr/bin/jimage:/usr/lib/jvm/openjdk14/bin/jimage
+ jdk:/usr/bin/jinfo:/usr/lib/jvm/openjdk14/bin/jinfo
+ jdk:/usr/bin/jjs:/usr/lib/jvm/openjdk14/bin/jjs
+ jdk:/usr/bin/jlink:/usr/lib/jvm/openjdk14/bin/jlink
+ jdk:/usr/bin/jmap:/usr/lib/jvm/openjdk14/bin/jmap
+ jdk:/usr/bin/jmod:/usr/lib/jvm/openjdk14/bin/jmod
+ jdk:/usr/bin/jps:/usr/lib/jvm/openjdk14/bin/jps
+ jdk:/usr/bin/jrunscript:/usr/lib/jvm/openjdk14/bin/jrunscript
+ jdk:/usr/bin/jshell:/usr/lib/jvm/openjdk14/bin/jshell
+ jdk:/usr/bin/jstack:/usr/lib/jvm/openjdk14/bin/jstack
+ jdk:/usr/bin/jstat:/usr/lib/jvm/openjdk14/bin/jstat
+ jdk:/usr/bin/jstatd:/usr/lib/jvm/openjdk14/bin/jstatd
+ jdk:/usr/bin/keytool:/usr/lib/jvm/openjdk14/bin/keytool
+ jdk:/usr/bin/pack200:/usr/lib/jvm/openjdk14/bin/pack200
+ jdk:/usr/bin/rmic:/usr/lib/jvm/openjdk14/bin/rmic
+ jdk:/usr/bin/rmid:/usr/lib/jvm/openjdk14/bin/rmid
+ jdk:/usr/bin/rmiregistry:/usr/lib/jvm/openjdk14/bin/rmiregistry
+ jdk:/usr/bin/serialver:/usr/lib/jvm/openjdk14/bin/serialver
+ jdk:/usr/bin/unpack200:/usr/lib/jvm/openjdk14/bin/unpack200
+"
+
+post_extract() {
+	chmod +x configure
+	if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
```

Comment:
Ideally, keep this consistent with prior XBPS_TARGET_LIBC checks, use `case` or `if` the whole way though.